lib/manager/tool-versions/extract.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { Fixtures } from '../../../test/fixtures';
+import { extractPackageFile } from './extract';
+
+describe('manager/tool-versions/extract', () => {
+  describe('extractPackageFile()', () => {
+    it('returns null for empty', () => {
+      expect(extractPackageFile('', '.tool-versions', {})).toBeNull();
+    });
+
+    it('extracts dependencies for supported tools', () => {
+      const res = extractPackageFile(
+        Fixtures.get('.tool-versions'),
+        'unused_file_name',
+        {}
+      );
+      expect(res).toEqual({
+        deps: [
+          {
+            currentValue: '10.15.0',
+            datasource: 'npm',
+            depName: 'nodejs',
+          },
+          {
+            currentDigest: 'v1.0.2-a',
+            currentValue: 'ref:v1.0.2-a',
```

Comment:
Are you sure that `ref` here points to pypi and not to git?

Reply:
Yes, this won't work. `pypi` datasource won't support those ref values. Should be something like:

```js
            currentValue: 'v1.0.2-a',
```

There is also no `python` package on pypi.org